crescent/commands/signature.py

```diff
@@ -31,17 +31,20 @@
 
 
 def _unwrap_optional(origin: type[Any]) -> Any:
+    args = get_args(origin)
+
+    if len(args) == 2:
+        if args[0] is NoneType:
+            return args[1]
+        if args[1] is NoneType:
+            return args[0]
+
     if get_origin(origin) is not Union:
         return origin
 
-    args = get_args(origin)
-
     if len(args) != 2 or NoneType not in args:
         return args
```

Comment:
Because of 36-40 this will always evaluate to true.

If the length of the args == 2, then we know that NoneType can't be in the args, as that would've been caught by 36-40.

If the length of args != 2, then, well, `len(args) != 2` passes.

Reply:
I do not understand after reading this like 10 times but tests pass so it lgtm